docs/team/hsiaotingluv.md

```diff
@@ -2,122 +2,80 @@
 layout: page
 title: Hsiao Ting's Project Portfolio Page
 ---
-### 1. Introduction
+### Project: Socius
 
-This is my project portfolio for Socius. The document outlines my contributions to the project, including the features and enhancements that I have implemented.
-
-#### 1.1 About the team
-
-Our team consists of Hsiao Ting, Choon Yong, Kevin, Boon Kee and Nathan. We are a group of Computer Science students from the National University of Singapore, and members of AY2022S1-CS2103T-W08-4 in the CS2103T Software Engineering module.
-
-#### 1.2 About the project
-
-The team project (tP) uses a generic application called [AddressBook-Level3 (AB3)](https://se-education.org) as the starting point.
-
-Accordingly, the tP is structured to resemble an intermediate stage of a non-trivial real-life brownfield software project in which we will,
-* conceptualize and implement enhancements to a given product, and,
-* have it ready to be continued by future developers
-
-Our team has decided to evolve AB3 into a more powerful or more optimized contact management app, Socius.
-
-Socius is a desktop application that can help you, as a CS2103T Software Engineering student, to
-* manage your classmates’ contacts
-* make friends and
-* find teammates!
-
-Socius is optimized for use via a *Command Line Interface (CLI)* while still having the benefits of a *Graphical User Interface (GUI)*.
-If you can type fast, Socius can get your contact management tasks done faster than traditional *GUI* apps.
-
-### 2. Summary of contributions
+Socius is a desktop application that can help you, as a CS2103T Software Engineering student, to manage your classmates’ contacts, make friends and find teammates!
+It is optimized for use via a CLI while still having the benefits of a GUI.
```

Comment:
Might want to consider using the standardised introduction to look more uniform. Can refer to any of our PPPs